WordPress/WordPressStatsWidgets/LockScreenWidgets/Views/LockScreenStatsWidgetsView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+import WidgetKit
+
+protocol LockScreenStatsWidgetsViewProvider {
+    associatedtype SiteSelectedView: View
+
+    @ViewBuilder
+    func buildSiteSelectedView(_ data: HomeWidgetData) -> SiteSelectedView
+
+    func statsURL(_ data: HomeWidgetData) -> URL?
+}
+
+struct LockScreenStatsWidgetsView<T: LockScreenStatsWidgetsViewProvider>: View {
+    let timelineEntry: StatsWidgetEntry
+    let viewProvider: T
+
+    @ViewBuilder
+    var body: some View {
+        switch timelineEntry {
+        case let .siteSelected(data, _):
+            viewProvider
+                .buildSiteSelectedView(data)
+                .widgetURL(viewProvider.statsURL(data))
+        default:
+            Text("Build Later")
```

Comment:
Are there further improvements coming for this "Build later" label in other PRs?

Reply:
Yes, other status will be implemented in the following PRs, I added TODO comments to avoid confusion.